CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+This document outlines how this team ships features, bug fixes, and everything
+in between.
+
+## Workflow
+
+The most common workflow is:
```

Comment:
Maybe rather than "most common", something like "Our workflow looks like this:"?